src/app.js

```diff
@@ -277,18 +275,23 @@ function buildBigScreenWidget(toolUrl) {
   </div>`;
 
   return {
-    appendTo,
+    insertAfter,
     html,
-    parentRelative,
   };
 }
 
 function buildSmallScreenWidget(toolUrl) {
   const appendTo = '#mobile-header';
-  let parentRelative = true;
+  const parentRelative = true;
+  let cssClass;
```

Comment:
do this: `let cssClass = '';`
Otherwise it could apply 'undefined' as a css class